zbus/src/abstractions/process.rs

```diff
@@ -19,6 +23,18 @@ impl Command {
         return Self(tokio::process::Command::new(program));
     }
 
+    /// Sets executable argument.
+    ///
+    /// Set the first process argument, `argv[0]`, to something other than the
+    /// default executable path.
+    pub fn arg0<S>(&mut self, arg: S) -> &mut Command
```

Comment:
This and other changes to `Command` (except for `spawn`) are fixup to the commit adding `Command` type. Please move them to that commit.